.github/workflows/eslint-styleline.yml

```diff
@@ -0,0 +1,42 @@
+name: eslint-stylelint
+
+on:
+  pull_request:
+    branches:
+      - master
+      - rhel-8
```

Comment:
Any reason why you are so picky here? `on: [pull_request]` should be quite enough, and then also cover future stable branches?